tests/lib/database/settings/structures/PermissionNodeManager.test.ts

```diff
@@ -19,7 +19,7 @@ describe('PermissionNodeManager', () => {
 		return entity.permissionNodes['sorted'];
 	}
 
-	describe('run', () => {
+	describe.skip('run', () => {
```

Comment:
TODO

Reply:
Those aren't even going to be used with slash commands. I'm skipping those because it's wasting a lot of time and energy, specially considering the system has proven itself from the +2 years it's been implemented and running in production.

Reply:
Then delete the code